src/components/services-view/services-view.html

```diff
@@ -1,8 +1,17 @@
 
 <div class="services-view">
 
-  <overlay-panel show-panel="$ctrl.orderingPanelvisible" show-close="true" handle-close="$ctrl.closeOrderingPanel">
-    <order-service service-class="$ctrl.serviceToOrder" handle-close="$ctrl.closeOrderingPanel"></order-service>
+  <overlay-panel show-panel="$ctrl.orderingPanelVisible" show-close="true" handle-close="$ctrl.closeOrderingPanel">
+    <order-service
+        ng-if="$ctrl.selectedServiceClass"
+        service-class="$ctrl.selectedServiceClass"
+        handle-close="$ctrl.closeOrderingPanel">
+    </order-service>
+    <builder-dialog
```

Comment:
create-app-from-builder is most accurate but does get a bit long, create-from-builder ?  create-app feels too generic, we already know new-app as a CLI command confuses people